### DISCUSSION TITLE
How to Use New Fabrication System

### DISCUSSION THREAD

**Discussion by Leboweeb** [MrTJP/ProjectRed#1821] [Q&A]
### Minecraft version

1.18.x

### CodeChickenLib version

4.1.4.488

### CBMultipart version

3.1.0.136

### ProjectRed version

4.16.0-beta-48

### What happened?

Modules Installed:
- Core
- Transmission
- Integration
- Fabrication
- Exploration

So far I've been figuring things out because the wiki is outdated and the fabrication process seems clear until you reach the packing table.

From what I understand, the pipeline is:

Blueprint -> Photomask -> Die 

Once I get the valid/invalid die how do I progress from there and use the packaging table? I put the die around the red square but how do I proceed from here? 
![IMG-20231225-WA0022](https://github.com/MrTJP/ProjectRed/assets/74187782/1796369d-b99c-4141-9932-d5ce885a92f6)

I apologize for using the bug template but I can't make an issue without using it for some reason. Also, If you want me to create a wiki and add documentation for the new system I'd be more than happy to do so.

### Crash log

.

### How do you make it happen again?

.

**Comment by MrTJP** [comment]
Hi. I locked down issue creation without a template to encourage using the Discussion tab for general questions. 

To use the packaging table, you put the valid dies in the middle slot, Plates in the corners, and the 4 remaining slots represent each IO type for the corresponding sides. If a side has no IO, use a plate. For bundled IO, use a Bundled Plate. RedStone IO gets Redstone plate. 

Note that currently, there is only 1 IO gate tile available in the editor (bundled). Will be adding Redstone soon, and a comparator-like sensing input later on. 

Let me know if you need more clarification. I’ll be creating a wiki once the system is more mature. Also welcome to suggestions to make the system more intuitive so external documentation is not needed.

**Reply by Leboweeb** [reply]
First of all, I'm sorry I reported this as an issue instead of using the discussion tab. However, I really wasn't joking when I said I'd at least help with documentation because your hands are already full as is. Perhaps I can submit a pull request for a few markdown files?
